src/main/java/ijopencv/examples/DistanceTransform.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * To the extent possible under law, the ImageJ developers have waived
+ * all copyright and related or neighboring rights to this tutorial code.
+ *
+ * See the CC0 1.0 Universal license for details:
+ *     http://creativecommons.org/publicdomain/zero/1.0/
+ */
+
+package ijopencv.examples;
+
+
+// import net.imglib2.type.numeric.RealType;
+
+import ij.gui.GenericDialog;
+
+import org.scijava.command.Command;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+
+import ij.IJ;
+import ij.ImagePlus;
+import ijopencv.ij.ImagePlusMatConverter;
+import ijopencv.opencv.MatImagePlusConverter;
+
+
+
+import static org.bytedeco.javacpp.opencv_imgproc.distanceTransform;
+import org.bytedeco.javacpp.opencv_imgproc;
+import org.bytedeco.javacpp.opencv_core;
+import org.bytedeco.javacpp.opencv_core.Mat;
+
+
+
+/**
+ * This example illustrates how to create an ImageJ {@link Command} plugin.
+ * <p>
+ * The code here is the DistanceTransform using ImageJ Ops.
+ * </p>
+ * <p>
+ * You should replace the parameter fields with your own inputs and outputs,
+ * and replace the {@link run} method implementation with your own logic.
+ * </p>
+ */
+@Plugin(type = Command.class, menuPath = "Plugins>IJ-OpenCV-plugins>DistanceTransform")
+public class DistanceTransform implements Command {
+    
+    String measure_method; // select the measurement method from openCV 
+    String speed; // select the correctness / speed for openCV
+    int threshold = 0; // select the threshold for 8bit gray, 0 means no threshold
+   
+    public DistanceTransform() {
+		// Auto-generated constructor stub
+	}
+    
+    
+    private boolean showDialog() {
+        GenericDialog gd = new GenericDialog("Distance Transform");
+        String[] items_measure = {"4Move", "8Move", "Euclidian"};
+        String[] items_speed = {"fast", "precise"};
+        gd.addChoice("Measure_Method", items_measure, items_measure[2]);
+        gd.addChoice("speed", items_speed, items_speed[1]);
+        gd.addNumericField("Threshold", threshold, 0);
```

Comment:
Yes, it can be done with `@Parameter`s. 

I have updated the code to work with `@Parameter`s since I understand it is the recommended option in the current version of ImageJ.